fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -531,6 +533,9 @@ public class SessionVariable implements Serializable, Writable {
     @VariableMgr.VarAttr(name = ENABLE_ODBC_TRANSCATION)
     public boolean enableOdbcTransaction = false;
 
+    @VariableMgr.VarAttr(name = ENABLE_RUN_SERIAL)
```

Comment:
Please use `description` field to add document for this new variable

Reply:
And better rename it to a more concrete one. `run serial` is too common

Reply:
done